flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -4307,15 +4307,20 @@
                 <NODE_ID>PUSH_ATC_FAIL_SEQ2</NODE_ID>
                 <EMISSIVE_CODE>(L:A32NX_OVHD_INTLT_ANN) 0 == (L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) and</EMISSIVE_CODE>
             </UseTemplate>
-
+            <UseTemplate Name="FBW_Anim_Interactions">
+                <ANIM_TEMPLATE>FBW_Airbus_ATC_Panel_ATC</ANIM_TEMPLATE>
+                <ANIM_TYPE>KNOB</ANIM_TYPE>
+                <Node_ID>Knob_ATC_1</Node_ID>
+                <PART_ID>Knob_ATC_1</PART_ID>
+            </UseTemplate>
```

Comment:
What is this one for? KNOB_ATC_1 is already covered by the main transponder template.